### DISCUSSION TITLE
Error TUNOVELASLIGERAS

### DISCUSSION THREAD

**Discussion by hamunoz** [lnreader/lnreader#967] [General]
Carga la lista de novelas en tunovelaligeras y acedes a una de las novelas, comienza a listar los capitulos o paginas  pero luego muestra un error de bypass cloudfare

**Comment by rajarsheechatterjee** [comment]
Create an issue in [LNReader/lnreader-sources](https://github.com/LNReader/lnreader-sources/issues/new?assignees=&labels=Source+Request&projects=&template=request_source.yml) repository.
